config/initializers/rollbar.rb

```diff
@@ -50,6 +50,10 @@
   # You can supply custom Sidekiq options:
   # config.use_sidekiq 'queue' => 'default'
 
+  # Don't report exceptions from retrying sidekiq jobs until third retry
+  # This is unrelated to and independent from `use_sidekiq` (though the docs are very unclear)
+  config.sidekiq_threshold = 3
```

Comment:
https://github.com/rollbar/rollbar-gem/blob/master/docs/configuration.md#sidekiq_threshold

> Ignored unless you've called `use_sidekiq`.

I think we should remove `sidekiq_threshold` again instead of enabling `use_sidekiq`.
I do not like how hard it is to figure out how [the sidekiq plugin](https://github.com/rollbar/rollbar-gem/blob/master/lib/rollbar/plugins/sidekiq/plugin.rb) works.
Also, it will probably be unexpected with this threshold. Meaning that jobs that are not retried (for instance, because they are cron jobs running every 5 minutes) will never get retried.

Reply:
I realize that I was the one championing for enabling this feature, and that jobs not retrying will get reported anyway...
So maybe we should enable `use_sidekiq` and keep the threshold. I can see both benefits and potential problems.